### PR TITLE
General code cleanup

### PR DESCRIPTION
**Pull request by maniac103**

### CODE REVIEW

**Comment by maniac103** [comment]
App code itself should be OK now; tests still have code style issues that need to be fixed.

File: mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.java [hunk 1]
```diff
@@ -887,7 +882,8 @@ public void onBackPressed() {
         Log.d(TAG, "onBackPressed()");
         if (mController.canGoBack()) {
             mController.goBack();
-        } else if (!isFullscreenEnabled()) { //in fullscreen don't continue back which would exit the app
+        } else if (!isFullscreenEnabled()) {
+            //in fullscreen don't continue back which would exit the app
```
Comment (mueller-ma) [inline mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.java]: Whitespace before `in`.

File: mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.java [hunk 2]
```diff
@@ -789,10 +795,11 @@ public void bind(OpenHABWidget widget) {
         public void onClick(final View view) {
             float minValue = mBoundWidget.minValue();
             float maxValue = mBoundWidget.maxValue();
-            //This prevents an exception below. But could lead to user confusion if this case is ever encountered.
+            //This prevents an exception below, but could lead to
```
Comment (mueller-ma) [inline mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.java]: Whitespace before `This`.

File: mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.java [hunk 3]
```diff
@@ -789,10 +795,11 @@ public void bind(OpenHABWidget widget) {
...
+            // user confusion if this case is ever encountered.
             float stepSize = minValue == maxValue ? 1 : mBoundWidget.step();
-
-            final String[] stepValues = new String[((int) (Math.abs(maxValue - minValue) / stepSize)) + 1];
+            final int stepCount = ((int) (Math.abs(maxValue - minValue) / stepSize)) + 1;
+            final String[] stepValues = new String[stepCount];
             for (int i = 0; i < stepValues.length; i++) {
                 //Check if step size is a whole integer.
```
Comment (mueller-ma) [inline mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.java]: Same here

**Pull request merged**